app/controllers/help_controller.rb

```diff
@@ -41,19 +41,23 @@ def create
     end
 
     if @support_form.valid?
-      template_id = GOV_NOTIFY_CONFIG['help_email']['template_id']
+      if ENV['ZENDESK_API_ENDPOINT'].present?
```

Comment:
Bit worried that this could mean that a miss-configuration go undetected, not sure we will look in the logs for this message.

If we remove this conditional will it blow up?

Reply:
Chatted about this briefly in real life - this conditional's basically for local dev. 

Misconfiguration will make it blow up later down the stack, where we're using `fetch` etc.